editor/js/editor.js

```diff
@@ -122,18 +122,32 @@
   // show the header
   header.classList.remove("hidden");
 
-  /* Initialise the editors. If there is a `dataset` property
-       of type `tabs` on the `editorContainer`, pass its value
-       to `initEditor` */
-  if (editorContainer.dataset && editorContainer.dataset.tabs) {
-    tabby.initEditor(
-      editorContainer.dataset.tabs.split(","),
-      document.getElementById("js")
-    );
-  } else {
-    tabby.initEditor(["html", "css"], document.getElementById("html"));
+  /**
+   * @returns {string[]} - IDs of editors that should be visible in the example.
+   */
+  function getTabs(editorContainer) {
+    if(editorContainer.dataset && editorContainer.dataset.tabs)
+      return editorContainer.dataset.tabs.split(",");
+    else
+      return ["html", "css"];
```

Comment:
For consistency, let's add a space after the `if` and use curly braces:

```suggestion
    if (editorContainer.dataset && editorContainer.dataset.tabs) {
      return editorContainer.dataset.tabs.split(",");
    } else {
      return ["html", "css"];
     }
```

(We should enforce this with `lint-staged`, but that's a separate issue.)